e2e/tests/cohorts/cohort-edit.spec.ts

```diff
@@ -39,34 +76,17 @@ describe('Editing Cohort tests', () => {
 
     await cohortBuildPage.discardChangesConfirmationDialog();
 
-    // Back to Data page.
+    // Discard changes brings user back to the Data page.
     await dataPage.waitForLoad();
-    const resourceCard = new DataResourceCard(page);
-    expect(await resourceCard.findCard(cohortName, ResourceCard.Cohort)).toBeTruthy();
   });
 
-  test('Delete cohort', async () => {
```

Comment:
`test('Delete cohort')` is removed. Because testing delete in Edit Cohort page is also covered in test below, `test('Save As another new cohort')`. So, I can safely remove this test while not lose any test coverage.